README.md

```diff
@@ -164,6 +164,7 @@ We are so excited to have you!
   [good first issues](https://github.com/shipwright-io/build/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
   and our
   [help wanted issues](https://github.com/shipwright-io/build/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+- Join our weekly community calls. See our public calendar [invite](https://calendar.google.com/calendar/u/1?cid=Y19iMWVndjc3anUyczJkbWNkM2R1ZnAxazhuNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t)
```

Comment:
* https://calendar.google.com/calendar/embed?src=c_b1egv77ju2s2dmcd3dufp1k8n4%40group.calendar.google.com&ctz=America%2FNew_York

* https://calendar.google.com/calendar/ical/c_b1egv77ju2s2dmcd3dufp1k8n4%40group.calendar.google.com/public/basic.ics

Do any of these links help? :(

Reply:
No, the summary of the item is just this: `SUMMARY:Busy` (from ical, same in the visualized HTML).

Reply:
Hi @mattcui ,

Can you take a look if you can add the meeting by using above link?
Thanks!

Reply:
@sbose78 I can host this meetings, therefore I can generate the proper calendar invite. What do you think? This PR is currently blocked because of the invites not being PUBLIC.